README.md

```diff
@@ -844,6 +845,9 @@ The following example of a proportion boolean measure shows how the logic highli
 
 ![Screenshot of Highlighting HTML Measure Logic](./static/logic-highlighting-example-2.png)
 
+### CQL Statement Ordering
+By default, the statements in the generated HTML output are sorted into population statements, non-functions, and functions, respectively. To disable this behavior, use the `disableHTMLOrdering` calculation option.
```

Comment:
You can decide if you think this is necessary or not, but in the PR for this functionality, Lauren said: "Generated HTML sorts statements into population statements, non-functions, and function statements, where non-population statements are also in alphabetical order". Maybe we want to add the thing about non-population statements being in alphabetical order?